nix/nixos/cardano-node-service.nix

```diff
@@ -404,6 +458,38 @@ in {
         description = ''Internal representation of the config.'';
       };
 
+      targetNumberOfRootPeers = mkOption {
+        type = types.int;
+        default = cfg.nodeConfig.TargetNumberOfRootPeers or 30;
```

Comment:
I think that a default value of 30 is too few root peers. I propose that it is increased to 60.
Given your formulas below it means that by default the node will have 20 active peers which is what we have assumed.